symengine/complex_mpc.cpp

```diff
@@ -908,6 +908,28 @@ class EvaluateMPC : public Evaluate
                 MPFR_RNDN);
         return complex_mpc(std::move(t));
     }
+    virtual RCP<const Basic> floor(const Basic &x) const override
+    {
+        SYMENGINE_ASSERT(is_a<ComplexMPC>(x))
+        mpc_class t(down_cast<const ComplexMPC &>(x).as_mpc().get_prec());
+        mpc_set_fr_fr(
```

Comment:
@isuruf What should be used here? I couldn't find `MPC`'s alternative to `floor` and `ceil`.